src/main/java/javapoz24/team3/rental/application/config/RentalId.java

```diff
@@ -0,0 +1,30 @@
+package javapoz24.team3.rental.application.config;
+
+// This singleton stores our Rental Id
+public class RentalId {
+    private static volatile RentalId instance;
+
+    private Long id;
+
+    private RentalId() {    }
+
+    public static RentalId getInstance() {
+        if (instance == null) {
+            synchronized (RentalId.class) {
```

Comment:
Nie wydaje mi się że blok synchrozied jest tu wymagany i if jest powtórzony

Reply:
W naszym przypadku nie będzie raczej wielu wątków, ale wzorzec lazy singleton przewiduje to zabezpieczenie.

Reply:
ok